backend/worker.py

```diff
@@ -285,6 +287,11 @@ def post(self):
                     entity_key=post_key,
                     message=notification_message
                 )
+                
+                if is_first_like:
+                    title = "Primeira curtida"
```

Comment:
It could be some kind of Enum. Notifications Types.

Reply:
I'm improving it in another branch